src/ipyvizzustory/storylib/story.py

```diff
@@ -197,11 +258,54 @@ def is_pixel(value: Any) -> bool:
             `True` if the value is pixel, `False` otherwise.
         """
 
-        value_is_pixel = False
-        if isinstance(value, str):
-            if value.endswith("px"):
-                value_is_pixel = value[:-2].isnumeric()
-        return value_is_pixel
+        if StorySize._is_int(value) or StorySize._is_float(value):
+            return True
+        if isinstance(value, str) and value.endswith("px"):
+            if StorySize._is_int(value[0:-2]) or StorySize._is_float(value[0:-2]):
+                return True
+        return False
+
+    def get_width_height_in_pixels(self) -> Tuple[int, int]:
+        """
+        A method for returning the width and height in pixels.
+
+        Raises:
+            ValueError: If width is not in pixels.
+            ValueError: If aspect_ratio is not set and height is not in pixels.
+
+        Returns:
+            The width and height in pixels as int.
+        """
+
+        if self.aspect_ratio is None:
+            if any(
+                [
+                    not StorySize.is_pixel(self.width),
+                    not StorySize.is_pixel(self.height),
+                ]
+            ):
+                raise ValueError("width and height should be in pixels")
+            _width = int(float(self.width[:-2]))  # type: ignore
+            _height = int(float(self.height[:-2]))  # type: ignore
+        else:
+            if not isinstance(self.aspect_ratio, (float, int)):
+                raise ValueError("aspect_ratio should be a float")
+            if all(
```

Comment:
Can you De Morgan-ize it?

all(not P1, not P2) --> not any(P1, P2)

It says that any of should be pixel, so it's simpler to understand the condition

Reply:
fixed in [323bebd](https://github.com/vizzuhq/ipyvizzu-story/pull/75/commits/323bebd16454db182989946f6ef1440d1715c1d0)